.github/workflows/docker.yml

```diff
@@ -23,12 +29,93 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
+    - name: Log in to the GH Container registry
+      uses: docker/login-action@v1.12.0
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}    
+   
     - name: Set up environment
       run: |
         export GOPATH=$(go env GOPATH)
         echo "GOPATH=$GOPATH" >> $GITHUB_ENV
         echo "$GOPATH/bin" >> $GITHUB_PATH
 
-    - name: Build images
-      run: |
-        make -C dist/images all
+    - name: Build ovnkube-binaries copy to context 
```

Comment:
I dont think we want to do what this commit is doing on every PR. Adds CI overhead and a bunch of image creation that probably no one is going to use. I think you want to do this on merge right? Or maybe periodically.

Also the github workflow for this:
```on:
  push:
  pull_request:
    branches: [ master ]
    paths: [ "dist/images/Dockerfile*"]
```
does that indicate it will only run when the file path is modified?

Reply:
I think you need to remove the entire pull_request block, so images are only created on push/merge.

Reply:
+1, fixed this to  only run anytime any code is pushed to master, regardless of PATH